lib/puppet/resource/capability_finder.rb

```diff
@@ -35,15 +35,10 @@ def self.find(environment, code_id, cap)
           ['select_catalogs',
             ['=', 'code_id', code_id]]]]
     end
-    query = query_terms.to_json
 
-    Puppet.notice "Capability lookup #{cap}]: #{query}"
+    Puppet.notice "Capability lookup #{cap}]: #{query_terms}"
```

Comment:
It looks basically the same either way (except for spaces), so it seemed like there was no need to JSONify it just for consistency with the old behavior. Could always change it if need be.

Before:
```
["and",["=","type","Sql"],["=","title","foo"],["=","tag","producer:production"],["in","certname",["extract","certname",["select_catalogs",["=","code_id","123456"]]]]]
```

After:
```
["and", ["=", "type", "Sql"], ["=", "title", "foo"], ["=", "tag", "producer:production"], ["in", "certname", ["extract", "certname", ["select_catalogs", ["=", "code_id", "123456"]]]]]
```